spec/draft-barnes-cfrg-4q.md

```diff
@@ -25,43 +25,44 @@ author:
        email: watsonbladd@gmail.com
 
 informative:
-    EFD:
-      target: "https://hyperelliptic.org/EFD/"
-      title: "Explicit-Formulas Database"
-      author:
-        -
-              ins: D.J. Bernstein
-        -
-              ins: T. Lange
- 
+
     Curve4Q:
       target: "https://eprint.iacr.org/2015/565.pdf"
-      title: "FourQ:four-dimensional decompositions on a Q-curve over the Mersenne prime"
+      title: "FourQ: four-dimensional decompositions on a Q-curve over the Mersenne prime"
       author:
          -
-              ins: C. Costello
+              ins: Craig Costello
          -
-              ins: P. Longa
+              ins: Patrick Longa    
+              
+    FourQlib:
+      target: "https://www.microsoft.com/en-us/research/project/fourqlib/"
+      title: "FourQlib"
+      author:
+         -
+              ins: Craig Costello
+         -
+              ins: Patrick Longa
                 
     TwistedRevisited:
        target: "http://iacr.org/archive/asiacrypt2008/53500329/53500329.pdf"
        title: Twisted Edwards Curves Revisited
        author:
           -
-              ins: H. Hisil
+              ins: Huseyin Hisil
           -
-              ins: K.H.Wong
+              ins: Kenneth Koon-Ho Wong
           -
-              ins: G. Carter
+              ins: Gary Carter
           -
-              ins: E. Dawson
+              ins: Ed Dawson
             
     Twisted:
-       target: "http://iacr.org/archive/"
+       target: "http://eprint.iacr.org/2008/013.pdf"
        title: Twisted Edwards Curves
        author:
           -
-              ins: D.J. Bernstein
+              ins: Daniel J. Bernstein
```

Comment:
I'm not familiar with the convention neither. But whatever we use it should be consistent across all the references. Feel free to adjust it.